src/connectors/aws_inventory.py

```diff
@@ -405,6 +483,33 @@ def ingest_elb(landing_table, aws_access_key=None, aws_secret_key=None, session=
     )
     return len(elbs)
 
+def get_iam_users(aws_access_key=None, aws_secret_key=None, session=None, account=None):
+    client = boto3.client(
```

Comment:
seems like this client is unused, you can remove it

Reply:
We still need to initialize it before the if statement right?